pages/main.js

```diff
@@ -113,6 +124,16 @@ function mainView (state, prev, send) {
   `
 }
 
+function WelcomeScreen (methods) {
+  const onExit = methods.onexit
```

Comment:
I chose to not use camelcase here to keep the method names similar to `lib/modal` which in turn is similar to `onload` which in turn is similar to the built-in DOM events